### PR TITLE
Webpack setup

### PR DESCRIPTION
**Pull request by zman811**
I set up the webpack as well as a very basic file layout. Also, set up eslint to use the airbnb style guide. 
Not totally sure that the webpack is set up fully if someone wants to take a look and see if there is anything I missed that would be great.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "my-webpack-project",
```
Comment (slargman) [inline package.json:2]: Maybe we can change this to our project name?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "version": "1.0.0",
+  "description": "My webpack project",
```
Comment (slargman) [inline package.json:4]: We could add a description here too.

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+// Generated using webpack-cli https://github.com/webpack/webpack-cli
+
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const isProduction = process.env.NODE_ENV == 'production';
```
Comment (slargman) [inline webpack.config.js:6]: should we use triple equals here?

**Review by slargman — CHANGES_REQUESTED** [review]
Looks good! We could fill in some of the information specific to our project on our package.json file though.

**Comment by slargman** [comment]
I'm not sure what all we will need to setup webpack either, but this looks like a good start. If anything isn't working once we're trying to get the app up and running we can adjust from there.

**Comment by zman811** [comment]
Cool sounds good, ill go update that now

**Review by slargman — APPROVED** [review]
Perfect. Thanks for taking care of this Zach!

**Pull request merged**